pkg/services/services.go

```diff
@@ -194,12 +190,57 @@ func klogVerbosityArg(debug bool) string {
 }
 
 // getKubeletHostnameOverride returns the hostname override arg that should be used
-func getKubeletHostnameOverride(platformType string) string {
+func getKubeletHostnameOverride(platformType string) (string, servicescm.PowershellCmdArg) {
```

Comment:
This addresses https://github.com/openshift/windows-machine-config-operator/pull/1260#discussion_r997390403